components/support/migration/src/main/java/mozilla/components/support/migration/FennecSessionMigration.kt

```diff
@@ -36,13 +36,21 @@ internal object FennecSessionMigration {
 
         for (file in sessionFiles) {
             try {
-                return parseJSON(file.readText())
+                return StreamingSessionStoreParser.parse(file)
+            } catch (e: FileNotFoundException) {
+                logger.error("FileNotFoundException while trying to parse sessopm stpre", e)
```

Comment:
You're not submitting this exception because we expect to see it, or..? Maybe change the logger level to `warn` to indicate that?

Reply:
Yeah, I think it is expected that the store or its backup do not exist. Agree, warning makes sense here.